### ISSUE TITLE
JSON Scripts

### ISSUE THREAD

**Issue by geertberkers** [Nictiz/Nictiz-STU3-testscripts#1]
Is it also possible to upload all test scripts in JSON format?

**Comment by ArdonToonstra** [comment]
Hi Geert,

Recently, we made the PHR testricts more generic. They are now capable to handle JSON requests as well. The testscript itself doesn’t have to be JSON for that.

This week or in the beginning of next week, these updated testcripts will be transferred to the testing folder so that they can be used within Touchstone for all MedMij vendors.

Does this positively answer your question?

Kind regards,
Ardon

**Comment by geertberkers** [comment]
Not really, we wrote our own FHIRClient, which currently only handles JSON.
I want to parse the TestScripts into our own FHIRClient, so it is possible for us to automate some tests.

**Comment by ArdonToonstra** [comment]
Out of primarily governance / maintenance reasons we don’t like to have a duplicate test scripts in different formats. I would suggest to convert the test scripts yourself.

Kind regards,
Ardon
